dask/base.py

```diff
@@ -850,9 +850,10 @@ def tokenize(*args, **kwargs):
     >>> tokenize('Hello') == tokenize('Hello')
     True
     """
+    hasher = md5(str(tuple(map(normalize_token, args))).encode())
     if kwargs:
-        args = args + (kwargs,)
-    return md5(str(tuple(map(normalize_token, args))).encode()).hexdigest()
+        hasher.update(str(normalize_token(kwargs)).encode())
```

Comment:
it might it be worth copying the test case from https://github.com/dask/dask/issues/8632 ?